packages/design-system/src/components/typography/link/index.js

```diff
@@ -79,6 +80,21 @@ export function Link({ children, ...props }) {
     </StyledAnchor>
   );
 }
+
+const LinkWithRef = forwardRef(function LinkWithRef(
+  { children, ...props },
+  ref
+) {
+  const isExternalLink = props.target === '_blank';
+  return (
+    <StyledAnchor ref={ref} {...props}>
+      <ConditionalSpanWrapper isWrapped={isExternalLink}>
+        {children}
+        {isExternalLink && <StyledLaunch />}
+      </ConditionalSpanWrapper>
+    </StyledAnchor>
+  );
+});
```

Comment:
💭 This is the same component as `Link`. is there a reason why we're exporting one that is wrapped with `forwardRef` and one that isn't?

If there isn't a reason I think we should just export one:

```js
export function Link = forwardRef(function Link(
  { children, ...props },
  ref
) {
  const isExternalLink = props.target === '_blank';
  return (
    <StyledAnchor ref={ref} {...props}>
      <ConditionalSpanWrapper isWrapped={isExternalLink}>
        {children}
        {isExternalLink && <StyledLaunch />}
      </ConditionalSpanWrapper>
    </StyledAnchor>
  );
});
```

Reply:
you're absolutely right. I had it this way originally because it seemed like a bunch of console warnings and test failures were associated with it, but it's fine now. Thank you for flagging this.